tests/test_hashstore.py

```diff
@@ -71,15 +71,14 @@ def test_rel_path(pids, store):
     assert path.endswith("7052d7e166017f779cbc193357c3a5006ee8b8457230bcf7abcef65e")
 
 
-def test_sysmeta_cid(store):
+def test_retrieve(store):
```

Comment:
The test should test all of the files in the test harness, not just one.

Reply:
Thanks Matt - I revised the test to check all the files in the test harness.